components/detail.vue

```diff
@@ -62,18 +62,22 @@
             :title="$t(`details.vending.${point.properties.tags.vending}`)"
             icon="osm-vending_machine"
           />
-          <detail-link
-            v-if="contact('phone')"
-            :href="`tel:${contact('phone')}`"
-            :title="contact('phone')"
-            icon="osm-phone"
-          />
-          <detail-link
-            v-if="contact('mobile')"
-            :href="`tel:${contact('mobile')}`"
-            :title="contact('mobile')"
-            icon="osm-mobile_phone"
-          />
+          <template v-if="contact('phone')">
+            <detail-link
+              v-for="phone in contact('phone').split(';')"
```

Comment:
t'a pas un warning parce qu'il manque un `key` ?

Reply:
où ?

Reply:
Dans les logs du serveur:
> (Emitted value instead of an instance of Error) <detail-link v-for="phone in contact('phone').split(';')">: component lists rendered with v-for should have explicit keys. See https://vuejs.org/guide/list.html#key for more info